conf/configuration.yaml

```diff
@@ -148,6 +149,7 @@ arlas_stac:
   description: ${ARLAS_STAC_DESCRIPTION:-Catalog for Arlas Exploration items}
   conforms_to:
     - https://api.stacspec.org/v1.0.0/core
+    - https://api.stacspec.org/v1.0.0-beta.2/core
```

Comment:
I think we should not merge this PR until we understand why add this line fix the CI